src/utils/cdk.test.ts

```diff
@@ -0,0 +1,194 @@
+import { CodeMaker } from 'codemaker'
+import { Config } from './args'
+import { CdkBuilder, CDKTemplate } from "./cdk"
+import { Imports, importType } from './imports'
+
+const mockedCodeMaker: {[key: string]: jest.Mock} = {
+  line: jest.fn(),
+  openBlock: jest.fn(),
+  closeBlock: jest.fn(),
+  indent: jest.fn(),
+  unindent: jest.fn(),
+}
+
+const clearMockedCodeMaker = (): void => {
+  Object.keys(mockedCodeMaker).forEach(key => mockedCodeMaker[key].mockClear())
+}
+
+describe("The CdkBuilder class", () => {
+
+  describe("addImport function", () => {
+    const builder = new CdkBuilder({} as Config, new Imports(), {} as CDKTemplate)
+    builder.code = mockedCodeMaker as unknown as CodeMaker
+
+    beforeEach(clearMockedCodeMaker)
+
+    afterEach(() => {
+      builder.imports = new Imports()
+    })
+
+    test("pads the imports section with new lines", () => {
+      builder.addImports()
+      expect(mockedCodeMaker.line.mock.calls[0]).toEqual([])
+      expect(mockedCodeMaker.line.mock.calls[2]).toEqual([])
+    })
+
+    test("always adds the @aws-cdk/core library", () => {
+      builder.addImports()
+      expect(mockedCodeMaker.line.mock.calls[1][0]).toEqual(`import cdk = require("@aws-cdk/core")`)
+    })
+
+    test("adds ALL style imports correctly", () => {
```

Comment:
We dropped this ability in our recent update to the linting rules; do we want to keep it here?